docs/README.md

```diff
@@ -14,7 +14,7 @@ The Hipster Shop application consists of a number of microservices, written in a
 
 ![image](./images/user-guide/1-architecture.png)
 
-**Note:** We are not endorsing this architecture as the best way to build a real online store. This application is optimized for demonstration and learning purposes.  It illustrates a large number of cloud-native technologies, uses a variety of programming languages, and provides an environment that can be explored productively with Ops Management tools.
+**Note:** You are not endorsing this architecture as the best way to build a real online store. This application is optimized for demonstration and learning purposes.  It illustrates a large number of cloud-native technologies, uses a variety of programming languages, and provides an environment that can be explored productively with Ops Management tools.
```

Comment:
**We** (i.e. the authors/maintainers of Sandbox) are not endorsing this architecture